User Guide/Projects.md

```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
 - SSL Client certificate
 - SSL Private key 
 - Private key password - you will receive this in your email.
+- Click Download certificate, pop-up to download the certs in .zip, enter a password to save the file in your local.
```

Comment:
"save the file in your local." -> "save the zip file in your computer."

Reply:
ok KC updated.